### DISCUSSION TITLE
Stop streaming

### DISCUSSION THREAD

**Discussion by siemprepe** [fuatakgun/eufy_security#556] [Q&A]
Hey,

Probably a dumb question, but how do you stop streaming? I clicked the camera snapshot and it switched to streaming (which is nice), but how do I stop the streaming. It stays like this forever?

**Comment by siemprepe** [comment]
Anyone please? I am unable to stop the stream and this also stops any notification from working. Turning the camera on or off, reloading the integration etc doesn't change the streaming status.

**Comment by lankymat** [comment]
Hi, not sure if you have found the answer yet but if you set it up as per the instructions you need to create a button (or call the stop service another way) which stops the camera.  If you go to the very bottom of the readme.md/webpage there is an example code of a conditional card.  Setting it up this way, when your feed starts to show it will also show a stop streaming button which you should press to halt the stream.  All the button does is call the camera.turn_off service.  Hope that helps and makes sense.

**Comment by siemprepe** [comment]
Thanks,
Seems I indeed missed something in the documentation. RTFM right :).

Unfortunatley I am now facing a different issue where the card for the stream is saying: Can't get URL for camera.xxx
